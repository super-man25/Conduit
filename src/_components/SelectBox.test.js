import React from 'react';
import { SelectBox } from './';
import renderer from 'react-test-renderer';

it('renders correctly with no props', () => {
  const tree = renderer
    .create(<SelectBox />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with noBg prop', () => {
  const tree = renderer
    .create(<SelectBox noBg />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with valid prop', () => {
  const tree = renderer
    .create(<SelectBox valid />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with inValid prop', () => {
  const tree = renderer
    .create(<SelectBox inValid />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
