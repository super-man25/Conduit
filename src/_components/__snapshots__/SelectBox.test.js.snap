// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with inValid prop 1`] = `
<select
  className="sc-jDwBTQ dxWESe"
/>
`;

exports[`renders correctly with no props 1`] = `
<select
  className="sc-jDwBTQ goylHR"
/>
`;

exports[`renders correctly with noBg prop 1`] = `
<select
  className="sc-jDwBTQ fglLhX"
/>
`;

exports[`renders correctly with valid prop 1`] = `
<select
  className="sc-jDwBTQ jcuSbL"
/>
`;
